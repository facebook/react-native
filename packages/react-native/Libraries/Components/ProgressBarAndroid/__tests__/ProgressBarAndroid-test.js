/**
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @flow strict-local
 * @format
 */

'use strict';

const ReactNativeTestTools = require('../../../Utilities/ReactNativeTestTools');
// $FlowFixMe[missing-platform-support]
const ProgressBarAndroid = require('../ProgressBarAndroid.android').default;
const React = require('react');

describe('<ProgressBarAndroid />', () => {
  it('should render as expected', async () => {
    await ReactNativeTestTools.expectRendersMatchingSnapshot(
      'ProgressBarAndroid',
      () => <ProgressBarAndroid styleAttr="Horizontal" indeterminate={true} />,
      () => {
        jest.dontMock('../ProgressBarAndroid');
      },
    );
  });
});
