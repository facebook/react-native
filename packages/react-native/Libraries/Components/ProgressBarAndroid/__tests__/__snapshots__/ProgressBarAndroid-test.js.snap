// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProgressBarAndroid /> should render as expected: should deep render when mocked (please verify output manually) 1`] = `
<AndroidProgressBar
  animating={true}
  indeterminate={true}
  styleAttr="Horizontal"
/>
`;

exports[`<ProgressBarAndroid /> should render as expected: should deep render when not mocked (please verify output manually) 1`] = `
<AndroidProgressBar
  animating={true}
  indeterminate={true}
  styleAttr="Horizontal"
/>
`;
